blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/inputs/handlers/HttpHandlerIntegrationTest.java

```diff
@@ -52,6 +64,12 @@
     //A time stamp 2 days ago
     private final long baseMillis = Calendar.getInstance().getTimeInMillis() - 172800000;
 
+    @Before
+    public void setup() throws Exception {
+        super.esSetup();
+        ((EventElasticSearchIO) eventsSearchIO).setClient(getClient());
```

Comment:
It seems odd to set this into a field which looks like is in the super class. This might be happening elsewhere and this one just caught my eye.

Reply:
I didn't touch this code. I wanted to make sure whatever minimum is *needed* to get this upgraded, I should do that. I should come back to this in later iteration.